go.mod

```diff
@@ -416,3 +418,9 @@ replace oras.land/oras-go => oras.land/oras-go v1.2.4-0.20230801060855-932dd06d3
 // testcontainers-go has a bug with versions v0.25.0 and v0.26.0
 // ref: https://github.com/testcontainers/testcontainers-go/issues/1782
 replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.23.0
+
+replace github.com/aquasecurity/trivy-policies => /Users/simarpreetsingh/repos/trivy-policies
+
+replace github.com/aquasecurity/trivy-aws => /Users/simarpreetsingh/repos/trivy-aws
+
+replace github.com/aquasecurity/defsec => /Users/simarpreetsingh/repos/defsec
```

Comment:
Update when https://github.com/aquasecurity/trivy-policies/pull/39 and https://github.com/aquasecurity/trivy-aws/pull/55 have merged.